src/components/ebay-textbox/README.md

```diff
@@ -0,0 +1,44 @@
+# ebay-textbox
+
+## ebay-textbox Usage
+
+Default input textbox:
+
+```marko
+<ebay-textbox />
+```
+
+When input textbox is disabled:
+
+```marko
+<ebay-textbox disabled/>
+```
+
+When input textbox has default placeholder:
+
+```marko
+<ebay-textbox placeholder="enter value" />
+```
+
+When input textbox fills up the containing block or fluid:
+
+```marko
+<ebay-textbox fluid />
+```
+
+Default multi-line textbox:
+
+```marko
+<ebay-textbox multiline />
+```
+
+## ebay-textbox Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`class` | String | No | define custom class
+`placeholder` | String | No |
+`fluid` | Boolean | No | defaults to false
+`disabled` | Boolean | Yes | defaults to false
```

Comment:
All of our boolean attributes default to `false`, so no need to mention. Can describe the behavior of the `true` case for these instead.